packages/maskbook/src/plugins/ITO/hooks/useFillCallback.ts

```diff
@@ -207,10 +214,10 @@ export function useFillCallback(poolSettings?: PoolSettings) {
             total,
             limit,
             DEFAULT_QUALIFICATION_ADDRESS,
-        ]
+        ] as Parameters<ITO['methods']['fill_pool']>
```

Comment:
Ok so I want to point out,

```ts
const x: T = ...
```

Is much type safe than

```ts
const x = ... as T
```

Reply:
Fixed.